packages/react-storage/src/components/StorageBrowser/actions/handlers/__tests__/listLocations.spec.ts

```diff
@@ -1,7 +1,117 @@
-import { listLocationsHandler } from '../listLocations';
+import {
+  listCallerAccessGrants,
+  ListLocationsOutput,
+  LocationCredentialsProvider,
+} from '../../../storage-internal';
+import {
+  listLocationsHandler,
+  ListLocationsHandlerInput,
+} from '../listLocations';
+import { LocationAccess } from '../types';
+import { parseLocations } from '../utils';
+
+jest.mock('../../../storage-internal');
+
+const mockListCallerAccessGrants = jest.mocked(listCallerAccessGrants);
+
+const generateMockLocations = (size: number, mockLocations: LocationAccess) =>
+  Array<LocationAccess>(size).fill(mockLocations);
 
 describe('listLocationsHandler', () => {
-  it('is null', () => {
-    expect(listLocationsHandler).toBeNull();
+  const accountId = 'account-id';
+  const credentials: LocationCredentialsProvider = () =>
+    Promise.resolve({
+      credentials: {
+        accessKeyId: 'access-key',
+        secretAccessKey: 'secret-key',
+        sessionToken: 'session-token',
+        expiration: new Date(),
+      },
+    });
+  const region = 'region';
+  const bucket = 'bucket';
+  const DEFAULT_PAGE_SIZE = 5;
+
+  const input: ListLocationsHandlerInput = {
+    config: { accountId, credentials, region, bucket },
+    options: {
+      pageSize: DEFAULT_PAGE_SIZE,
+      nextToken: undefined,
+      exclude: 'READ',
+    },
+    prefix: 'prefix',
+  };
+
+  const id = 'intentionally-static-test-id';
```

Comment:
Doesn't look like we are asserting against `id`, can it just be returned directly on line 48?